spec/emerald_spec.rb

```diff
@@ -0,0 +1,129 @@
+require 'spec_helper'
+require 'tempfile'
+require 'helper/user_comms'
+include Emerald
```

Comment:
This was originally left over from when I was trying to test this class and couldn't `require` it because it was expecting command line arguments. I'm wondering whether it's worth including it now though, so I can access its classes directly e.g. ```UserCommsHelper::InvalidFileTypeError``` - Is this a bad idea though, to open up the possibility of classes including `Emerald`. Could it result in having multiple references to it, and potentially altering its behaviour?